Sources/SKSwiftPMWorkspace/SwiftPMWorkspace.swift

```diff
@@ -10,6 +10,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+import SPMBuildCore
```

Comment:
Can you make this:

```
 #if canImport(SPMBuildCore)
import SPMBuildCore
#endif
```
This will allow us to avoid rev-lock and land this first.

Reply:
Done!